HomeWork/ACoder.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace HomeWork
+{
+    public class ACoder : ICoder
+    {
+        public string Decode(string str)
+        {
+            if(str is null)
+            {
+                throw new ArgumentNullException("Строка не должна быть Null.");
```

Comment:
```suggestion
                throw new ArgumentNullException(nameof(str), "Строка не должна быть Null.");
```
Первым параметром указывается название параметра